.github/AUTORELEASE.md

```
@@ -0,0 +1,69 @@
+- `VILNACRM_APP_PRIVATE_KEY`: Add the private key you generated earlier.
+- `VILNACRM_APP_ID`: Add the app ID you copied.
+
+#### 3) Allow force push
+
+To ensure the GitHub App can perform necessary actions:
+
+1. Go to Settings > Branches.
+2. In the branch protection rules, check the option to "Allow force pushes".
+3. Specify that the only allowed actor is the GitHub app you installed.
+
+Note: Force pushing allows overwriting the Git history. It's restricted to the GitHub App to maintain security while allowing necessary operations for the auto-release workflow.
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)